### ISSUE TITLE
Rocket / boost mode

### ISSUE THREAD

**Issue by Doogie13** [bebeli555/ElytraHighwayBot#8]
Overworld only mode to use rockets to move around in overworld (should bypass 2b2t.org)
Also a rocketless mode which gains altitude with boost (akin to the boost bypass in konas and kami blue for 2b2t)
Would make the mod wildly more popular with 2b2t players who currently cannot afk elytra and the bypass would draw in new users (even though its public).

**Comment by bebeli555** [comment]
Im working on a new mod rn that will include the elytrabot and with firework support. The boost mode doesnt really work on 2b2t as it closes your elytra after a bit and then you cant reopen it anymore for like 15 seconds.
